beestation.dme

```diff
@@ -2232,6 +2232,7 @@
 #include "code\modules\client\preferences\entries\character\jobless_role.dm"
 #include "code\modules\client\preferences\entries\character\names.dm"
 #include "code\modules\client\preferences\entries\character\pda.dm"
+#include "code\modules\client\preferences\entries\character\quirkoptions.dm"
```

Comment:
This file should follow standard naming conventions. Be sure to rename the actual file as well.
```suggestion
#include "code\modules\client\preferences\entries\character\quirk_options.dm"
```